### DISCUSSION TITLE
Command+F keyboard shortcut makes it impossible to search for text in the page

### DISCUSSION THREAD

**Discussion by olafurpg** [community/community#5825] [Projects and Issues]
I'm sorry if this is a duplicate of another discussion, I tried to find a related discussion and was surprised this wasn't already actively being discussed. Please don't use `Cmd+F` as a custom keyboard shortcut because it hijacks the browser's `Cmd+F` functionality, which is essential to find information on the open page.

**Comment by yilei** [comment]
+1, `Cmd+F` should always be reserved as browser's built-in search functionality.

**Comment by toddhodgson** [comment]
I hate this "feature" too. Can you at least allow us to disable this?

**Comment by cillianderoiste** [comment]
In Firefox, you can disable all browser shortcut overrides for a particular website as described here: https://support.mozilla.org/en-US/kb/firefox-page-info-window#w_override-keyboard-shortcuts
